flow/connectors/clickhouse/clickhouse.go

```diff
@@ -77,24 +77,43 @@ func (c *ClickHouseConnector) ValidateCheck(ctx context.Context) error {
 	err := c.database.Exec(ctx, fmt.Sprintf(`CREATE TABLE IF NOT EXISTS %s (
 		id UInt64
 	) ENGINE = ReplacingMergeTree ORDER BY id;`,
```

Comment:
```suggestion
	) ENGINE = Memory ORDER BY id;`,
```
Are permissions engine specific?

Reply:
unsure on this, will leave as is for now